alembic-autogenerate.py

```diff
@@ -3,27 +3,28 @@
 #
 # How to use this script:
 # 1. Load the LSST environment and setup sdm_schemas and felis.
-#        source loadLSST.bash
+#        source loadLSST.bash or activate your eups setup
 #        setup felis
-#        setup -r /path/to/sdm_schemas
+#        setup -r /path/to/sdm_schemas -- SDM_SCHEMAS_DIR should work
 # 2. From the root of the consdb git repo, invoke the script. Supply a
 #    revision message as the command line argument:
 #        python alembic-autogenerate.py this is my revision message "\n" \
```

Comment:
I wouldn't think that we want this to be a really long message as in this example. 

I recommend we use a ticket number here (e.g., `DM-12345`), which is then automatically appended to the generated Python script.

Reply:
That makes sense, and you have implemented that in the auto generation PR.